test-packages/e2e-tests/package.json

```diff
@@ -12,9 +12,9 @@
     "start": "pm2 start server.js || pm2 restart server.js",
     "stop": "pm2 stop server.js",
     "deploy": "cds deploy --to sqlite",
-    "test": "yarn jest --runInBand",
-    "test:local": "yarn jest",
-    "check:dependencies": "depcheck . --ignores='express,sqlite3,@sap/cds'"
+    "test": "yarn pretest && yarn root:jest --runInBand && yarn posttest",
```

Comment:
why all this pretest and post is needed? Is this not executed autotically?

Reply:
If so I am afraid we missed it somewhere else.

Reply:
Only arbitrary pre-&post scripts are no longer running by default, [life-cycle scripts](https://yarnpkg.com/advanced/lifecycle-scripts) however still work - this was the only occurrence of a script that isn't part of the lifecycle